flake.nix

```diff
@@ -95,6 +95,12 @@
           debug-tools = (pkgs.callPackage ./debug-tools.nix {
           });
 
+          develop = (pkgs.callPackage ./develop.nix {
```

Comment:
Why we can't just add  flag for enable`BUILD_CRYPTO3_TESTS` and `BUILD_PARALLEL_CRYPTO3_TESTS` to proof-producer derivation? I mean add `proof-producer-all-crypto3-test` instead adding new file

Reply:
done